packages/@aws-cdk/aws-codebuild/lib/build-spec.ts

```diff
@@ -8,6 +9,15 @@ export abstract class BuildSpec {
     return new ObjectBuildSpec(value);
   }
 
+  /**
+   * Export a YAML from an object
+   *
+   * @param value object to be exported to YAML
+   */
+  public static yamlFromObject(value: {[key: string]: any}): BuildSpec {
```

Comment:
I know I originally suggested this name, but looking at it today, I don't really like it 😕.

Apologies for the churn, but would you mind changing it to `fromObjectToYaml()`? I'm also open to suggestions for other names.

Reply:
it is fine, `fromObjectToYaml` sounds good :+1: 

Renamed it